.circleci/make-rpm-packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "PARAM_RELEASE: $PARAM_RELEASE"
+echo "PARAM_DIST: $PARAM_DIST"
+
+if test "x$PARAM_RELEASE" = "x"; then
+    echo "Env variable PARAM_RELEASE is not set! For example PARAM_RELEASE=8, for CentOS 8"
+    exit 1
+elif test "x$PARAM_DIST" = "x"; then
+    echo "Env variable PARAM_DIST is not set! For example PARAM_DIST=centos"
+    exit 1
+fi
+
+if [ "$PARAM_DIST" = centos ]; then
+  if [ "$PARAM_RELEASE" = 8 ]; then
+      dnf install -y 'dnf-command(config-manager)'
+      yum config-manager --set-enabled PowerTools
+      yum install -y diffutils python3-sphinx
+  else
+      yum install -y python-sphinx
+  fi
+fi
+
+yum install -y epel-release
+yum install -y automake jemalloc-devel git libedit-devel libtool libunwind-devel make pcre-devel python3 sudo rpm-build yum-utils
+
+
+export DIST_DIR=build
+
+# XXX: we should NOT have to do that here, they should be in the
+# spec as BuildRequires
+yum install -y make gcc
```

Comment:
let's open a PR on https://github.com/varnishcache/pkg-varnish-cache/ to fix it once and for all

Reply:
I will work on this now!

Reply:
Any idea what this error means:
```
+ /usr/lib/rpm/redhat/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-python-bytecompile /usr/bin/python 1
Compiling /root/rpmbuild/BUILDROOT/varnish-20200409.weekly-0.0.el7.x86_64/usr/share/varnish/vmodtool.py ...
  File "/usr/share/varnish/vmodtool.py", line 231
    print("ERROR: " + txt, file=sys.stderr)
                               ^
SyntaxError: invalid syntax

error: Bad exit status from /var/tmp/rpm-tmp.1T9NKA (%install)
```
https://app.circleci.com/pipelines/github/martin-g/varnish-cache/210/workflows/ba13bd1f-5cc6-42b7-8c1d-429632ad96ce/jobs/1354

Reply:
The syntax is OK with Python3, but fails with Python2.
It is because I commented out this line:
```
sed -i '1 i\%global __python %{__python3}' "$DIST_DIR"/redhat/varnish.spec
```
I'll revert this change and leave it for later

Reply:
https://github.com/varnishcache/pkg-varnish-cache/pull/138

The changes in Varnish Cache are at: https://github.com/martin-g/varnish-cache/pull/1/files
They can be merged to this PR only after https://github.com/varnishcache/pkg-varnish-cache/pull/138